CODEOWNERS

```diff
@@ -147,7 +147,6 @@ Kconfig*                                  @tejlmand
 /samples/app_event_manager/               @pdunaj @MarekPieta
 /samples/event_manager_proxy/             @rakons
 /samples/gazell/                          @leewkb4567
-/samples/keys/random_hw_unique_key/       @oyvindronningstad @Vge0rge
 /samples/keys/identity_key_generation/    @frkv @Vge0rge
 /samples/keys/identity_key_usage/         @frkv @Vge0rge
```

Comment:
Maybe just simplify this to a '/samples/keys/' entry?

Reply:
I didn't quite catch what you want to say here, could you clarify?   I removed the sample, which entry I can simplify?

Reply:
The two below, samples/keys can be one entry instead of one per sample.